### PR TITLE
EG-1068, refactored Callout implementaion

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/services/PageBuilder.scala [hunk 1]
```diff
@@ -129,7 +129,7 @@ class PageBuilder extends ProcessPopulation {
   def fromPageDetails[A](pages: Seq[Page])(f: (String, String, String) => A): List[A] =
```
Comment (woollett-r) [inline app/services/PageBuilder.scala:129]: If this method is only used by PageBuilderSpec I would move it into the test class.

Comment (cmrhmrc) [reply app/services/PageBuilder.scala:129]: This is used in the backend (PageBuilder shared between viewer and backend)

File: test/models/ui/StackingSpec.scala [hunk 2]
```diff
@@ -23,62 +23,63 @@ import models.ocelot.stanzas._
 class StackingSpec extends BaseSpec {
 
   trait Test {
-    def stackedCo(typ: CalloutType, stack: Boolean) = Callout(typ, Phrase(), Seq.empty, stack)
+    def stackedTitle(stack: Boolean) = TitleCallout(Phrase(), Seq.empty, stack)
```
Comment (woollett-r) [inline test/models/ui/StackingSpec.scala:26]: Can you add the type on these two methods to shut up Intellij please?

Comment (cmrhmrc) [reply test/models/ui/StackingSpec.scala:26]: done

**Pull request merged**